charts/spv-wallet-stack/charts/spv-wallet/templates/configmap.yml

```diff
@@ -8,91 +8,100 @@ data:
-        skip_initialize_with_version: {{  .Values.config.db.sql.skip_initialize_with_version }} 
-        table_prefix: {{  .Values.config.db.sql.table_prefix }} 
-        time_zone: {{  .Values.config.db.sql.time_zone }} 
-        tx_timeout: {{  .Values.config.db.sql.tx_timeout }} 
-        user: {{  .Values.postgresql.auth.username }}
-    debug: {{  .Values.config.debug }} 
-    debug_profiling: {{  .Values.config.debug_profiling }} 
-    disable_itc: {{  .Values.config.disable_itc }}
-    nodes:
-    {{- if .Values.config.nodes.deployment_id }}
-      deployment_id: {{ .Values.config.nodes.deployment_id }}
-    {{- else }}
-      deployment_id: 'spv-wallet-{{ printf "%s-$s"  (include "spv-wallet.fullname" .) .Values.global.domainName | adler32sum }}'
-    {{- end }}
+        debug: {{ .Values.config.db.sql.debug }}
+        driver: {{ .Values.config.db.sql.driver }}
+        host: "{{ .Release.Name }}-postgresql-spv-wallet"
+        max_connection_idle_time: {{ .Values.config.db.sql.max_connection_idle_time }}
+        max_connection_time: {{ .Values.config.db.sql.max_connection_time }}
+        max_idle_connections: {{ .Values.config.db.sql.max_idle_connections }}
+        max_open_connections: {{ .Values.config.db.sql.max_open_connections }}
+        name: {{ .Values.postgresql.auth.database }}
+        password: {{ .Values.config.db.sql.password }}
+        port: {{ .Values.config.db.sql.port }}
+        replica: {{ .Values.config.db.sql.replica }}
+        skip_initialize_with_version: {{ .Values.config.db.sql.skip_initialize_with_version }}
+        table_prefix: {{ .Values.config.db.sql.table_prefix }}
+        time_zone: {{ .Values.config.db.sql.time_zone }}
+        tx_timeout: {{ .Values.config.db.sql.tx_timeout }}
+        user: {{ .Values.postgresql.auth.username }}
+      sqlite:
+        database_path: {{ .Values.config.db.sqlite.database_path }}
+        debug: {{ .Values.config.db.sqlite.debug }}
+        max_connection_idle_time: {{ .Values.config.db.sqlite.max_connection_idle_time }}
+        max_connection_time: {{ .Values.config.db.sqlite.max_connection_time }}
+        max_idle_connections: {{ .Values.config.db.sqlite.max_idle_connections }}
+        max_open_connections: {{ .Values.config.db.sqlite.max_open_connections }}
+        shared: {{ .Values.config.db.sqlite.shared }}
+    debug: {{ .Values.config.debug }}
+    debug_profiling: {{ .Values.config.debug_profiling }}
+    disable_itc: {{ .Values.config.disable_itc }}
+    arc:
+      url: {{ .Values.config.arc.url }}
+      token: {{ .Values.config.arc.token }}
+      _deployment_id: {{ .Values.config.arc._deployment_id }}
       callback:
-        callback_host: 'https://{{.Values.global.domainName}}'
-      use_fee_quotes: {{ .Values.config.nodes.use_fee_quotes }}
+        enabled: {{ .Values.config.arc.callback.enabled }}
+        host: {{ .Values.config.arc.callback.host }}
+        _token: {{ .Values.config.arc.callback._token }}
```

Comment:
it should be rather optionally added, something like in pseudocode below:
```
if (.Values.config.arc.callback.token) {
  token: .Values.config.arc.callback._token
}
```

Bear in mind that the `_` shouldn't be used, because we use it in example config to show example but not use that static value, so the valid key is really without `_`